public/index.html

```diff
@@ -82,7 +82,7 @@
         pointer-events: none;
         width: 200vw;
         height: 200vh;
-        background: radial-gradient(50% 50% at 50% 50%, #fc077d10 0%, rgba(255, 255, 255, 0) 100%);
+        background: linear-gradient(180deg, #202738 0%, #070816 100%);
```

Comment:
@vm not sure how to feature flag this one

Reply:
you can't